### ISSUE TITLE
Argument 1 passed to Analogue\\ORM\\System\\EntityBuilder::build() must be of the type array, object given

### ISSUE THREAD

**Issue by 18601673727** [analogueorm/analogue#112]
While I do:

```
$this->articleMapper->query()->whereCid(1)->get();
```

caused this error:

```
Got error 'PHP message: PHP Catchable fatal error:  Argument 1 passed to Analogue\\ORM\\System\\EntityBuilder::build() must be of the type array, object given, called in /app/vendor/analogue/orm/src/System/Query.php on line 761 and defined in /app/vendor/analogue/orm/src/System/EntityBuilder.php on line 73
```

So I changed `vendor/analogue/orm/src/System/Query.php` on Line 755

from

```
$results = $this->query->get($columns);
```

to

```
$results = $this->query->get($columns)->all();
```

Problems solved, any suggestions?

**Comment by RemiCollin** [comment]
Which is the version of Laravel ?

**Comment by 18601673727** [comment]
I didn't use any frameworks in my current project sir.

**Comment by 18601673727** [comment]
composer.json
```
"Analogue/ORM": "^5.2",
"illuminate/support": "^5.2"
```

**Comment by RemiCollin** [comment]
This issue happens with the 5.3 version of laravel. What is your 'minimum-stability' in composer.json ? it really looks like it installed `illuminate/databse:5.3` (or `dev-master`)

Anyway, this will be fixed in next Analogue release (5.3) which i'm working on.

**Comment by RemiCollin** [comment]
Fixed in 5.3 release
